src/iOS/Avalonia.iOS/ClipboardImpl.cs

```diff
@@ -1,34 +1,63 @@
+#if !TVOS
```

Comment:
Can we just remove the file via a conditional `<Compile Remove` ?

Reply:
Done